tests/parm/nems.configure.hafs_atm_ocn_wav_inline.IN

```diff
@@ -0,0 +1,168 @@
+##############################################
+#####  NEMS Run-Time Configuration File  #####
+##############################################
+
+# ESMF #
+logKindFlag:            ESMF_LOGKIND_MULTI_ON
+globalResourceControl:  true
+
+# EARTH #
+EARTH_component_list: MED ATM OCN WAV
+EARTH_attributes::
+  Verbosity = 0
+::
+
+# MED #
+MED_model:                      @[med_model]
+MED_petlist_bounds:             @[med_petlist_bounds]
+MED_omp_num_threads:            @[med_omp_num_threads]
+MED_attributes::
+  Verbosity = 0
+  Diagnostic = 0
+  ATM_model = @[atm_model]
+  OCN_model = @[ocn_model]
+  WAV_model = @[wav_model]
+  MED_model = cmeps
+  history_n = 6
+  history_option = nhours
+  history_ymd = -999
+  coupling_mode = @[CPLMODE]
+  normalization = none
+  merge_type = copy
+  pio_rearranger = @[pio_rearranger]
+::
+
+# ATM #
+ATM_model:                      @[atm_model]
+ATM_petlist_bounds:             @[atm_petlist_bounds]
+ATM_omp_num_threads:            @[atm_omp_num_threads]
+ATM_attributes::
+  Verbosity = 0
+  Diagnostic = 0
+  mesh_atm = @[MESH_ATM]
+  stop_n = @[FHMAX]
+  stop_option = nhours
+  stop_ymd = -999
+  write_restart_at_endofrun = .true.
+::
+
+# OCN #
+OCN_model:                      @[ocn_model]
+OCN_petlist_bounds:             @[ocn_petlist_bounds]
+OCN_omp_num_threads:            @[ocn_omp_num_threads]
+OCN_attributes::
+  Verbosity = 0
+  Diagnostic = 0
+  cdf_impexp_freq = 3
+  cpl_hour = 0
+  cpl_min = 0
+  cpl_sec = @[coupling_interval_sec]
+  base_dtg = @[SYEAR]@[SMONTH]@[SDAY]@[SHOUR]
+  merge_import = .true.
+  skip_first_import = .true.
+  hycom_arche_output = .false.
+  hyc_esmf_exp_output = .true.
+  hyc_esmf_imp_output = .true.
+  import_diagnostics = .false.
+  import_setting = flexible
+  hyc_impexp_file = nems.configure
+  espc_show_impexp_minmax = .true.
+  ocean_start_dtg = @[OCEAN_START_DTG]
+  start_hour = 0
+  start_min = 0
+  start_sec = 0
+  end_hour = @[FHMAX]
+  end_min = 0
+  end_sec = 0
+::
+
+# WAV #
+WAV_model:                      @[wav_model]
+WAV_petlist_bounds:             @[wav_petlist_bounds]
+WAV_omp_num_threads:            @[wav_omp_num_threads]
+WAV_attributes::
+  Verbosity = 0
+  Diagnostic = 0
+  OverwriteSlice = false
+  merge_import = .true.
```

Comment:
I believe you'll want this setting to be .false. Otherwise, the wave cap will read the binary forcing data and probably over-write what is being supplied via the cdeps inline? 

In wav_import_export, if merge_import is true, an import mask will be created in set_importmask (using the special value) and then that mask is used to merge the binary data w/ the coupled import field.

Reply:
@uturuncoglu this is something that we've seen as well, but I've not seen the wave model fail due to this lack of first input of wind in the wave model

Reply:
@DeniseWorthen @JessicaMeixner-NOAA BTW, the interaction between -wave is happening in the atm_post phase after running the components. So, CDEPS inline does not fill the data in atm->wav direction since post is not run yet. So, it seems it is something like chicken and egg problem. @JessicaMeixner-NOAA It is failing if you set  `merge_import` to `false` since wave getting very large values from mediator and wave component does not overwrite the data coming from atm.

Reply:
Just to clarify, the issue about possibly over-writing what is getting sent via the new proposed CDEPS+CMEPS inline capability. I think the current cap, which reads a staged binary wind file for the non-overlapped region is working as expected.

Reply:
@DeniseWorthen I think there is no harm to activate those flags if we put extra control and make them work if they sees very large values in the incoming field. This works like that in the FV3 and if I remember me and Dan implemented like that in WW3 and HYCOM but we need to double check again.

Reply:
@DeniseWorthen @JessicaMeixner-NOAA @binli2337 @BinLiu-NOAA 

For HYCOM we have something like following, https://github.com/NOAA-EMC/HYCOM-src/blob/35789c757766e07f688b4c0c7c5229816f224b09/NUOPC/hycom_couple.F90#L733 It seems that the HYCOM only set its internal data structure if the value is not equal to fill value (same with the one that is set in CMEPS). But I could not see the same for other variables like `taux10` which is a coupling field (`tauy10` has one). So, maybe the implementation is not complete and need to be fixed. 

I think following line needs to be changed in WW3 mesh cap, https://github.com/NOAA-EMC/WW3/blob/97e6a63ebf9a9030fcdae6ad5cf85a0bc91fa37f/model/src/wav_import_export.F90#L381 since it reads all the data from file. It needs to read the file and only update the part if has fill value.

Reply:
@DeniseWorthen BTW, there is a `imp_merge` variable in the HYCOM cap but not used. Anyway, I think we just need to fix the components first. I am not sure how this was working before especially the missing check in the `taux10`. Maybe I am missing something in here. Any idea @BinLiu-NOAA @danrosen25?

Reply:
@uturuncoglu With merge_import=true, on the first timestep, it makes the mask=0 everywhere (so that only the file data is used) and then the second timestep, it sets the mask=0 using the large value. The mask is constant at that point but if merge-import is true, it will always be reading the binary global data and merging it using the mask.

Reply:
@DeniseWorthen I think WW3 needs to do same even if merge_import=false for the first coupling time step. Right? Then it will start getting complete field from mediator. I think it would be hard to do anything about it in the mediator level since post phase does not run yet.

Reply:
Leaving aside the issue of the first timestep for now, I don't think we want to use merge-import true because we don't want to be reading the binary file and making the merge in the cap. We'll be getting complete fields from the CMEPS+CDEPS implmentation, right? There will be no need to do the merge in the cap.

For the first timestep, when the fields are zero from the ATM, I don't think you want to rely on what the cap does currently, because that means you'd always have to have this binary file created and available. 

I'm guessing WW3 would be fine if 10m winds=0 on the first timestep, but what it can't deal with probably are "fill value" winds at the first timestep. So maybe we need the same sort of lagged-startup that we use in MOM6 (don't advance the first coupling timestep, back the clock up one timestep and advance two timesteps).

Reply:
@DeniseWorthen while waves  should be able to run with 0 winds for 1 time step, ideally we'd likely want to have some sort of initial value.  There's a chance that there's an operational application outside of whats being discussed here as the GEFS control member will likely be a cold start (cc'ing @NeilBarton-NOAA and @sbanihash for awareness).  For waves,  we'd either want to grab some information from the restart and/or do exactly as you say where we back up the clock one time step and then advance two timesteps.

Reply:
@uturuncoglu, @DeniseWorthen, @JessicaMeixner-NOAA, @junwang-noaa, I think for Ufuk's current work, from HAFS side, we would actually prefer to setting merge_import to true here to ensure the backward compatibility (at least at the moment for HAFSv2 upgrade perspective). The functionalities discussed above sound good to me. Meanwhile, we might can create an separate issue (or issues), and schedule some group discussions to figure out/choose an approach to develop/enable these capabilities through future PRs. Thanks!

Reply:
@BinLiu-NOAA When testing the new feature, how are you evaluating that it is working correctly if in effect you are over-writing the merged field from CMEPS+CDEPS with the data read from the file and merged locally in the cap?

Reply:
I am resolving this side the could support both `merge_import = .true.` and `merge_import = .false.` options in component side when CDEPS inline is activated. Under HAFS tests we are keeping `merge_import = .true.` for FV3ATM but merge_import = .false. for ocean and wave components.